### DISCUSSION TITLE
어느 한 3년차 프론트엔드 개발자의 2023년 회고

### DISCUSSION THREAD

**Discussion by giscus** [yiyb0603/yiyb-blog#51] [Blog Comment]
# 어느 한 3년차 프론트엔드 개발자의 2023년 회고

2023년 회고를 통해서 주요 일들을 정리합니다.

https://yiyb-blog.vercel.app/posts/2023-retrospect

**Comment by woojae05** [comment]
멋진 2023년을 보내셨네요!!

**Reply by yiyb0603** [reply]
좋게 봐주셔서 감사합니다!
